### ISSUE TITLE
transfer root_user role in deobs.users?

### ISSUE THREAD

**Issue by le9i0nx** [debops/ansible-console#28]
https://github.com/debops/ansible-console/blob/master/tasks/root_account.yml

do you think I should do it?
I will integrate into the role deobs.users
I'll try to keep backward compatibility.

**Comment by drybjed** [comment]
Well, hmm. That would be bringing back the `root` management code, since it originally started there and was [moved to debops.console in the first place](https://github.com/debops/ansible-users/commit/ff2c694234a75709bab502d731a1172bba92f07d).

Should it be moved back? On one hand roles like `debops.console` and `debops.auth` are currently sort of catch-all roles with many different functions bundled together. Splitting them into smaller roles would probably make a lot of sense, so that the functionality can be mixed and matched in the playbooks as needed.

The "theme" of the `debops.console` role are things that are usually important on the system console, or when you login to that system directly. By default you don't use the `root` password, unless you need to login to that account in case of recovery or local console access, at least in my experience. The `root` account is somewhat special and changing its configuration similarly how `debops.users` allows you to do with other users, might be dangerous and break things. That's why the role does not allow to modify most of the importnt bits like UID/GID, and the tasks themselves were moved to a different role to put the uncommon tasks from `debops.users` out of the way.

An alternative would be to put `root` management in its own separate role. That role would be small, but there are even smaller roles (like `debops.directories`), so that's not a deciding factor. Usually creating a separate role is done to create separate variable namespace which can then be expanded upon.

In conclusion, I'm not sure yet what should be done with this code. The `debops.console` role will probably be split into smaller ones, but I wouldn't start with this particular set of tasks. For example, creating separate role for `locales` seems to be more useful right now.

**Comment by le9i0nx** [comment]
thanks for the explanation.
